solutions_builder/template_root/terraform/stages/2-foundation/firestore_setup.tf

```diff
@@ -39,11 +39,11 @@ resource "google_project_service" "firestore" {
   service = "firestore.googleapis.com"
 }
 
-resource "google_firestore_database" "database" {
-  depends_on = [google_project_service.firestore]
+# resource "google_firestore_database" "database" {
```

Comment:
Why commenting out firestore

Reply:
The Firestore is able to create a default database without this block. I tested multiple times and it still behave like that.

Reply:
Updated with the logic, it only creates the Firestore when choosing a different name other than "(default)".